packages/storybook/src/executors/utils.ts

```diff
@@ -108,7 +108,9 @@ function reactWebpack5Check(options: CommonNxStorybookConfig) {
       encoding: 'utf8',
     });
 
-    if (!storybookConfig.includes(`builder: 'webpack5'`)) {
+    if (
+      !storybookConfig.match(/builder: ('webpack5'|"webpack5"|`webpack5`)/g)
```

Comment:
Could be `/builder: ['"`]webpack5['"`]/g` if you want to make it shorter.

Reply:
Yes, I thought about that, but then `"webpack5'` would be valid :/